src/arrayMethodJoin.js

```diff
@@ -3,9 +3,26 @@
 /**
  * Implement method join
  */
+
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    if (this.length === 0) {
```

Comment:
не потрібно перевіряти на 0
```suggestion
    if (!this.length) {
```